scripts/setup-app-layer.py

```diff
@@ -90,7 +90,7 @@ def patch_makefile_am(protoname):
     output = io.StringIO()
     with open("src/Makefile.am") as infile:
         for line in infile:
-            if line.startswith("app-layer-template.c"):
+            if line.startswith("	app-layer-template.c"):
```

Comment:
So before this break the line would look like:
```
app-layer-template.c app-layer-template.h
```
But now the headers are in a separate var, so this only applies to the .c file. Do you want to take a stab and fixing this up for the header as well?

Reply:
I was wondering about that difference, true! Sure, I like fixing things. Thanks for pointing that out! :)

Reply:
Should we rather add `lstrip` like `line.lstrip().startswith("app-layer-template.c"):` ?

Reply:
That sounds better, indeed. I wasn't sure my solution was good, just knew it needed some fixing...